### ISSUE TITLE
Can't open variables by clicking on the magnifiying glass icon

### ISSUE THREAD

**Issue by coldspark29** [lckr/jupyterlab-variableInspector#212]
I still can't open the variables in the inspector although I have all versions up-to-date. I already wrote you twice in the former issue, but I got no response, so I am opening an new one. Of course I am using numpy arrays.

**Comment by GabZech** [comment]
Are you sure the dataframe is not to big? When I try with smaller ones it works

**Comment by coldspark29** [comment]
Yes, I am pretty sure. It does not work with any array. For me the inspector is almost useless because of this.

**Comment by danfulton** [comment]
I appear to be having the same issue.  I'm not sure if I need to install some dependencies for this?  I did install numpy, but I wasn't sure if it should go in the environment running jupyter or in the kernel environment being inspected.
